distribution/docs/src/main/jdocs/content/_security-framework/security-soap-intro-contents.adoc

```diff
@@ -0,0 +1,88 @@
+| | Client |call endpoint | /---------\ |        |   |       |       |             |         |
+| |        |------------->| |  Read   | |        |   |       |       | Expansion   |         |
+| | cDEF   |              | | Policy  | |        |   |       |       | Service     |         |
+| |        |              | \---------/ |        |   |       |       |             |         |
+| \--------/              |      |      |        |   |       |       |             |         |
+|                         |      v      |        |   |       |       |             |         |
+|                         | /---------\ |        |   |       |       \-------------/         |
+|                         | |  Build  | |        |   |       |                               |
+|                         | |Security | |        |   |       |                               |
+|                         | | Headers | |        |   |       |                               |
+|                         | \---------/ |        |   |       |                               |
+|                         |      |      |        |   |       |                               |
+|                         |      v      |        |   |       |                               |
+|                         | /---------\ |        |   |       |                               |
+|                         | |   Get   | |        |   |       |                               |
+|                         | |Anon SAML| |        |   |       |                               |
+|                         | \---------/ |--------/   |       |                               |
+|                         |      | cDEF |            |       |                               |
+|                         |      v      |            |       |                               |
+|                         \-------------/            |       |                               |
+|                            ^   |    Anon Token     |       |                               |
+|                            |   \------------------>|       |                               |
+|                            |         Subject       |       |                               |
+|                            \-----------------------|       |                               |
+|                                                    \-------/                               |
+|                                                                                            |
++--------------------------------------------------------------------------------------------+
+....
+
+===== SOAP Secure Client
+
+When calling to an endpoint from a SOAP secure client, it first requests the WSDL from the endpoint and the SOAP endpoint returns the WSDL.
+The client then calls to STS for authentication token to proceed.
+If the client receives the token, it makes a secure call to the endpoint and receives results.
+
+===== Dumb SOAP Client
```

Comment:
Maybe we should say "Policy unaware SOAP Client" instead of "Dumb". It is dumb though ;)

Reply:
Updated.